File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        s = s.strip()

        if not s:
            return 0

        lis = list(s.split(" "))

        return len(lis[-1])

File: 0058-length-of-last-word/0058-length-of-last-word.php
class Solution {

    /**
     * @param String $s
     * @return Integer
     */
    function lengthOfLastWord($s) {
        $trimmedString = trim($s);
        $split = preg_split('/\s+/', $trimmedString);
        $lastWord = end($split);
        return strlen($lastWord);
    }
}
